_includes/google-search.html

```diff
@@ -0,0 +1,12 @@
+<script>
+    (function() {
+      var cx = 'e20a3e7ce37c2454c';
+      var gcse = document.createElement('script');
```

Comment:
I think we can assume ES6, so we can use `const` in place of `var` if you prefer.

Reply:
I tried the different function syntax and it broke the search bar (for some reason). I added the const.